### DISCUSSION TITLE
Complete Support for Yarn PnP

### DISCUSSION THREAD

**Discussion by NedcloarBR** [biomejs/biome#3393] [Ideas]
### Description

At the moment BiomeJS doesn't support Yarn PnP completely and this support is a [non-goal](https://github.com/biomejs/biome/issues/3391#issuecomment-2214981566) but in my view a large package used in different projects needs to have support for all popular package managers along with their ways of handling the packages
as a result, Biome is unable to resolve the shared configuration packages placed in the 'extends' property in the 'biome.json'

Probably similar: https://github.com/biomejs/biome/discussions/644

**Comment by zach-betz-hln** [comment]
I found this discussion after commenting on the linked issue. 

With a few extra steps, it's possible to use a shared biome config with yarn pnp. See this [comment](https://github.com/biomejs/biome/issues/3391#issuecomment-2663972301).

**Comment by ematipico** [comment]
Oxc resolver now supports PNP.

I'm not very fond of it, it would be nice to know what the other members think

**Reply by arendjr** [reply]
I think PnP is pretty great, but it has some real downsides too, especially when using it in combination with binary packages… such as Biome.

Overall I don’t mind having PnP support in Biome, but I don’t see it being much of a priority for us.
